src/CrossPointWebServer.cpp

```diff
@@ -0,0 +1,706 @@
+#include "CrossPointWebServer.h"
+
+#include <SD.h>
+#include <WiFi.h>
+
+#include <algorithm>
+
+#include "config.h"
+#include "html/FilesPageFooterHtml.generated.h"
+#include "html/FilesPageHeaderHtml.generated.h"
+#include "html/HomePageHtml.generated.h"
+
+// Global instance
+CrossPointWebServer crossPointWebServer;
```

Comment:
Instead of instantiating this as a global instance and listening to events as part of the main loop, can we remove this extern instance, move the `CrossPointWebServer` files/logic into `activities/network/server` and have it be instantiated by `CrossPointWebServerActivity`